include/odp/api/spec/traffic_mngr.h

```diff
@@ -2001,6 +2001,25 @@ int odp_tm_enq(odp_tm_queue_t tm_queue, odp_packet_t pkt);
 int odp_tm_enq_multi(odp_tm_queue_t tm_queue, const odp_packet_t packets[],
 		     int num);
 
+/** Send packets with segmentation offload to TM system
+ *
+ * Like odp_tm_enq_multi(), but segments packets according LSO configuration. See e.g.
+ * odp_pktout_send_lso() for documentation how packets are split into smaller packets during
+ * the segmentation offload. Packet segmentation is done first and TM functionality is applied
+ * to the resulting packets.
+ *
+ * @param tm_queue  Specifies the tm_queue (and indirectly the TM system)
+ * @param packets   Array of packets to enqueue with segmentation offload
+ * @param num       Number of packets in the array
+ * @param lso_opt   LSO options to be used for all packets. When NULL, LSO options are
+ *                  read from each packet (see odp_packet_lso_request()).
+ *
+ * @return Number of packets successfully processed and consumed (0 ... num)
+ * @retval <0 on failure
```

Comment:
I think this should be "<= 0 on failure" to match odp_tm_enq_multi(). Otherwise the API looks ok to me.

Reply:
Changed return value of enq_multi() in v5.